Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Core/MethodOverloaded.h

```diff
@@ -91,13 +93,15 @@ namespace ScriptCanvas
                 // \todo make execution thread sensitive, which can then support generic programming
                 Grammar::FunctionPrototype GetInputSignature() const;
 
+#if defined(OBJECT_STREAM_EDITOR_ASSET_LOADING_SUPPORT_ENABLED)////
                 // SerializeContextReadWriteHandler
-                void OnReadBegin();
-                void OnReadEnd();
-
-                void OnWriteBegin();
+                void OnReadBegin() {}
+                void OnReadEnd() {}
+                void OnWriteBegin() {}
                 void OnWriteEnd();
-                ////
+#endif//defined(OBJECT_STREAM_EDITOR_ASSET_LOADING_SUPPORT_ENABLED)
```

Comment:
Nitpick, repeated:  a space between `#endif` and the start of the comment would be more readable.